pytest_html/plugin.py

```diff
@@ -288,12 +290,14 @@ def generate_summary_item(self):
         summary = [html.h2('Summary'), html.p(
             '{0} tests ran in {1:.2f} seconds. '.format(
                 numtests, suite_time_delta)),
-            html.p('(Un)check the boxes to filter the results.')]
+            html.p('(Un)check the boxes to filter the results.',
+                   class_='filter',
+                   hidden=True)]
 
         for outcome in outcomes:
+            summary.append(' -')
             summary.append(outcome.checkbox)
             summary.append(outcome.summary_item)
-            summary.append(' ')
```

Comment:
This adds the separator at the beginning too, and misses to include a space after the dash. I would suggest the following:

```python
for i, outcome in enumerate(outcomes, start=1):
    summary.append(outcome.checkbox)
    summary.append(outcome.summary_item)
    if i < len(outcomes):
        summary.append(', ')
```